02week/tests.js

```diff
@@ -0,0 +1,75 @@
+//student:Jon Gorman
+//Assignment: Tests in rockPaperScissors
+//Instructor: Reneee Dudley
+//Date: 10/13/2017new change test
+//
+'use strict';
+
+const assert = require('assert');
+const readline = require('readline');
+const rl = readline.createInterface({
+    input: process.stdin,
+    output: process.stdout
+});
+
+
+function rockPaperScissors(a, b) {
+    // make variables that allow the function to pass the test ie. lowercase and whitespace.
+    const hand1 = a.toLowerCase().trim();
+    const hand2 = b.toLowerCase().trim();
+
+    //check for a tie
+    if (hand1 === hand2) {
+        return "It's a tie!";
+        // Write code here
+        // Check for hand one win
+    } else if (hand1 === 'rock' && hand2 === 'scissors' || hand1 === 'paper' && hand2 === 'rock' || hand1 === 'scissors' && hand2 === 'paper') {
+        return "Hand one wins!"
+        //Check for player two win
+    } else if (hand1 === 'rock' && hand2 === 'paper' || hand1 === 'paper' && hand2 === 'scissors' || hand1 === 'scissors' && hand2 === 'rock') {
+        return "Hand two wins!"
+    }else if (hand1 && hand2 !== 'rock', 'paper', 'scissors'){
```

Comment:
This is not how you check if hand1 and hand2 are not equal to rock or paper or scissors. It will probably throw an error.